src/cryptonote_core/tx_pool.cpp

```diff
@@ -466,7 +466,7 @@ namespace cryptonote
   }
   //---------------------------------------------------------------------------------
   //TODO: investigate whether boolean return is appropriate
-  bool tx_memory_pool::get_relayable_transactions(std::list<std::pair<crypto::hash, cryptonote::blobdata>> &txs) const
+  bool tx_memory_pool::get_relayable_transactions(std::list<std::pair<crypto::hash, cryptonote::blobdata>> &txs, bool include_unrelayed_txes) const
```

Comment:
I'm not sure the flag makes sense for that function.

Reply:
Currently not usefual as far as I can judge from my limited grasp of the code-base. Was more a future-proof stuff since it sounds like a function that could be used via RPC requests sooner or later.

I can remove to keep it simple.